### PR TITLE
153843248 Show GeoJSON in OTE

### PR DESCRIPTION
**Pull request by tatut**
**Make dialogs a little wider**

**Don't do anything in viewer start if not in CKAN**

**Open GeoJSON viewer directly without going to CKAN**

**Add stylefy styles for service viewer**

**Add close button**

**Fix reactive derefs**

### CODE REVIEW

File: ote/src/cljs/ote/app/controller/service_search.cljs [hunk 1]
```diff
@@ -68,4 +72,25 @@
...
+            :loading-geojson? true))
+
+  CloseServiceGeoJSON
+  (process-event [_ app]
+    (update app :service-search dissoc :resource :geojson :loading-geojson?))
+
+  GeoJSONFetched
+  (process-event [{response :response} app]
+    (update app :service-search assoc
+            :resource response
```
Comment (rckopteri) [inline ote/src/cljs/ote/app/controller/service_search.cljs]: :resource and :geojson keys both have the whole response as value.
Should this be optimised somehow?

File: ote/src/cljs/ote/style/service_viewer.cljs [hunk 2]
```diff
@@ -0,0 +1,25 @@
+(ns ote.style.service-viewer
+  "Styles for service viewer (GeoJSON view)"
+  (:require [stylefy.core :as stylefy]))
+
+(def properties-table {:text-align "left"
+                       ;;:border "solid 1px black"
```
Comment (rckopteri) [inline ote/src/cljs/ote/style/service_viewer.cljs]: commented code can be removed

File: ote/src/cljs/ote/style/service_viewer.cljs [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+                       })
+
+
+
+(def striped-even {:background-color "#f2f2f2"})
+(def striped-odd {:background-color "white"})
+
+(def striped-styles [striped-even striped-odd])
+
+(def border {}  #_{:border-bottom "solid #f1f1f1 1px"})
```
Comment (rckopteri) [inline ote/src/cljs/ote/style/service_viewer.cljs]: this one can be also removed

File: ote/src/cljs/ote/style/service_viewer.cljs [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+(def th (merge {:vertical-align "top"
+                :padding-top "0.2em"
+                :padding-right "0.5em"}
+               border))
```
Comment (rckopteri) [inline ote/src/cljs/ote/style/service_viewer.cljs]: merging is not needed

File: ote/src/cljs/ote/app/controller/service_search.cljs (hunk 1 above)
Comment (tatut) [reply ote/src/cljs/ote/app/controller/service_search.cljs]: The geojson (as a JS object) is required for Leaflet to draw it

File: ote/src/cljs/ote/style/service_viewer.cljs (hunk 2 above)
Comment (tatut) [reply ote/src/cljs/ote/style/service_viewer.cljs]: will do

**Pull request merged**